### PR TITLE
filtered movie list

### PR DESCRIPTION
**Pull request by kreisberg**
[DEMO LINK](https://kreisberg.github.io/react_movies-list-filter/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+  const handleOnChange = (event:React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setMovieName(value);
+  };
+
+  const visibleMoviesOnList = moviesFromServer.filter(movie => {
+    const title = movie.title.toLowerCase();
+    const description = movie.description.toLowerCase();
+    const name = movieName.toLowerCase().trim();
```
Comment (Sveta-Kryukova) [inline src/App.tsx:18]: You'd better rename this variable to something like 'preparedName' for better understanding what exactly name you mean)

**Review by Sveta-Kryukova — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (kreisberg) [reply src/App.tsx:18]: Thanks! Just fix it)

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [movieName, setMovieName] = useState('');
```
Comment (valerii-derkach) [inline src/App.tsx:7]: we check not only in movieName, but also in the description
```suggestion
  const [query, setQuery] = useState('');
```

File: src/App.tsx [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
+
+  const handleOnChange = (event:React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setMovieName(value);
+  };
+
+  const visibleMoviesOnList = moviesFromServer.filter(movie => {
```
Comment (valerii-derkach) [inline src/App.tsx:15]:
```suggestion
  const visibleMovies = moviesFromServer.filter(movie => {
```

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

**Review by valerii-derkach — APPROVED** [review]
🔥

**Pull request closed without merging**